docs/dev/table_api.md

```diff
@@ -1485,14 +1485,14 @@ Joins, set operations, and non-windowed aggregations are not supported yet.
 
 {% top %}
 
-### Group Windows
+### Windows
```

Comment:
I think we should stick to `Group Windows` here and rather adapt the description of the Table API windows. 
In SQL, the term `WINDOW` refers to the `OVER` windows, so I think it is less ambiguous if we call them `Group Windows` and `Over Windows`.